lib/resque/integration/tasks/hooks.rake

```diff
@@ -37,7 +37,7 @@ namespace :resque do
     end
 
     Resque.after_fork do |job|
-      $0 = "resque-#{Resque::Version}: Processing #{job.queue}/#{job.payload['class']} since #{Time.now.to_s(:db)}"
+      $0 = "resque-#{Resque::Version}: Processing #{job.queue}/#{job.payload['class']} since #{Time.now.to_s}"
```

Comment:
Lint/StringConversionInInterpolation: Redundant use of Object#to_s in interpolation.